packages/commonwealth/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
Is this safe????

Reply:
I think so. From my understanding the preserve option just keeps the jsx elements for some further compilation step to deal with. The react option converts them into React.createElements (which our webpack already does). This step is needed for tsmocha to not complain.

Anyways I think it would be best to test this on staging before we merge in order to make sure it doesn't break our production webpack build.

Reply:
Will resolve once confirmed on a production-like environment.